sdk/elastic/Azure.ResourceManager.Elastic/api/Azure.ResourceManager.Elastic.netstandard2.0.cs

```diff
@@ -142,24 +280,62 @@ protected MockableElasticResourceGroupResource() { }
+        public static Azure.ResourceManager.Elastic.Models.OpenAIIntegrationProperties OpenAIIntegrationProperties(string openAIResourceId = null, string openAIResourceEndpoint = null, string openAIConnectorId = null, string key = null, System.DateTimeOffset? lastRefreshOn = default(System.DateTimeOffset?)) { throw null; }
+        public static Azure.ResourceManager.Elastic.OpenAIIntegrationRPModelData OpenAIIntegrationRPModelData(Azure.Core.ResourceIdentifier id = null, string name = null, Azure.Core.ResourceType resourceType = default(Azure.Core.ResourceType), Azure.ResourceManager.Models.SystemData systemData = null, Azure.ResourceManager.Elastic.Models.OpenAIIntegrationProperties properties = null) { throw null; }
+        public static Azure.ResourceManager.Elastic.Models.OpenAIIntegrationStatusResponse OpenAIIntegrationStatusResponse(string status = null) { throw null; }
+        public static Azure.ResourceManager.Elastic.Models.PartnerBillingEntity PartnerBillingEntity(string id = null, string name = null, System.Uri partnerEntityUri = null) { throw null; }
+        public static Azure.ResourceManager.Elastic.Models.SubscriptionList SubscriptionList(Azure.ResourceManager.Elastic.Models.Operation? operation = default(Azure.ResourceManager.Elastic.Models.Operation?), System.Collections.Generic.IEnumerable<Azure.ResourceManager.Elastic.Models.MonitoredSubscription> monitoredSubscriptionList = null, Azure.ResourceManager.Elastic.Models.ProvisioningState? provisioningState = default(Azure.ResourceManager.Elastic.Models.ProvisioningState?)) { throw null; }
+        public static Azure.ResourceManager.Elastic.Models.UpgradableVersionsList UpgradableVersionsList(string currentVersion = null, System.Collections.Generic.IEnumerable<string> upgradableVersions = null) { throw null; }
+        public static Azure.ResourceManager.Elastic.Models.UserApiKeyResponse UserApiKeyResponse(string apiKey = null) { throw null; }
         public static Azure.ResourceManager.Elastic.Models.VmIngestionDetailsResponse VmIngestionDetailsResponse(string cloudId = null, string ingestionKey = null) { throw null; }
         public static Azure.ResourceManager.Elastic.Models.VmResources VmResources(string vmResourceId = null) { throw null; }
     }
+    public partial class BillingInfoResponse : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Elastic.Models.BillingInfoResponse>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Elastic.Models.BillingInfoResponse>
```

Comment:
We should not have a type ending with `Response`.
Could we rename this to something like `ElasticBillingInfoResult`?